packages/internals/src/cli/getSchema.ts

```diff
@@ -256,11 +192,15 @@ async function getSchemaWithPathInternal(
     return customSchemaResult
   }
 
+  // 2. Try the `schema` from `PrismaConfig`
```

Comment:
Guess that part is left for me to add? ;)

Reply:
Yess